Whisky/Views/Bottle/Pins/PinCreationView.swift

```diff
@@ -105,7 +94,8 @@ struct PinCreationView: View {
                 submit()
             }
         }
-        .frame(minWidth: 400, minHeight: 170)
+        .fixedSize(horizontal: false, vertical: true)
+        .frame(minWidth: 400)
```

Comment:
This has a lot more changes than just fixing the pin creation view. Please separate those out.